keps/sig-api-machinery/4420-retry-generate-name/kep.yaml

```diff
@@ -0,0 +1,39 @@
+title: Retry Generate Name
+kep-number: 4420
+authors:
+  - "@jpbetz"
+owning-sig: sig-apimachinery
+participating-sigs:
+status: implementable
+creation-date: 2024-01-19
+reviewers:
+  - TODO
+approvers:
+  - "@deads2k"
+
+# The target maturity stage in the current dev cycle for this KEP.
+stage: alpha
+
+# The most recent milestone for which work toward delivery of this KEP has been
+# done. This can be the current (upcoming) milestone, if it is being actively
+# worked on.
+latest-milestone: "v1.30"
+
+# The milestone at which this feature was, or is targeted to be, at each stage.
+milestone:
+  alpha: "v1.30"
+  beta: "v1.31"
+  stable: "v1.32"
+
+# The following PRR answers are required at alpha release
+# List the feature gate name and the components for which it must be enabled
+feature-gates:
+  - name: RetryGenerateName
```

Comment:
Gates describe the feature, framed as if it were already stable. Here's what I'd call it:
```yaml
  - name: NameGenerationRetries
```

Reply:
SGTM. Thx!